TAScheduler/AcceptanceTests/admin_create_course_test.py

```diff
@@ -0,0 +1,60 @@
+from django.test import TestCase
+from django.urls import reverse
+from TAScheduler.models import User, Course, Instructor, TA, Administrator
+
+
+class AdminCreateCourseTestCase(TestCase):
+    def setUp(self):
```

Comment:
Add lines for all variables used in setup and set them to null above the setUp method